docs/wiki/Home.md

```diff
@@ -81,13 +81,8 @@ NOS3 isn't a substitute for hardware testing, just a tool to augment the number
 The component development flow includes the development of a standalone checkout application that can more rapidly be developed, deployed, and tested on hardware.
 The flight software will use the same functions and code developed and tested through that process allowing focus to shift to data flow and system level tests cleanly.
 
-### What can NOS3 give me during Integration and Testing (I&T) ?
+### What can NOS3 give me during Integration and Testing (I&T)?
 
 During I&T activities, NOS3 enables you to perform integration testing of your satellite system, validate the operational procedures, and verify the overall system performance.
 NOS3 allows you to conduct end-to-end simulations, test different mission scenarios, and assess the behavior of the satellite system under various conditions.
 
-### How to a move from software / simulator to hardware once it arrives?
-
-NOS3 isn't a substitute for hardware testing, just a tool to augment the number of types of tests possible.
-The component development flow includes the development of a standalone checkout application that can more rapidly be developed, deployed, and tested on hardware.
-The flight software will use the same functions and code developed and tested through that process allowing focus to shift to data flow and system level tests cleanly.
```

Comment:
Should we be removing this bit?

Reply:
I added a sentence explaining how NOS3 can be used in hardware testing.